EssentialFeed/EssentialFeed/HTTPClient/Helpers/HTTPURLResponse+StatusCode.swift

```diff
@@ -6,8 +6,13 @@ import Foundation
 
 extension HTTPURLResponse {
 	private static var OK_200: Int { return 200 }
+	private static var OK_2XX: ClosedRange<Int> { return 200...299}
```

Comment:
Missing space

```suggestion
	private static var OK_2XX: ClosedRange<Int> { return 200...299 }
```

Spacing and indentation seem broken in many places. Make sure it's consistent everywhere following the codebase convention.